test/verify/check-subscriptions

```diff
@@ -47,6 +47,17 @@ import os
 # register with: activation key "awesome_os_pool" and org "admin"
 # or: subscription-manager register --activationkey awesome_os_pool --org admin --serverurl https://$IP:8443/candlepin
 
+WAIT_SCRIPT = """
+set -ex
+for x in $(seq 1 20); do
+    if curl --insecure -s https://%(addr)s:8443/candlepin; then
+        break
+    else
+        sleep $x
```

Comment:
I still think the linear backoff is unnecessary, especially since the server was already started with `systemctl start tomcat`, which blocks until the service is actually running. But if you really want to do this, I could live with it.